PWGCF/FemtoDream/FemtoUtils.h

```diff
@@ -9,59 +9,67 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-/// \file CFFilter.cxx
+/// \file FemtoUtils.h
 /// \brief Utilities for the FemtoDream framework
-///
 /// \author Luca Barioglio, TU München, luca.barioglio@cern.ch
 
 #ifndef PWGCF_FEMTODREAM_FEMTOUTILS_H_
 #define PWGCF_FEMTODREAM_FEMTOUTILS_H_
 
-#include <CCDB/BasicCCDBManager.h>
 #include <vector>
 #include "Framework/ASoAHelpers.h"
 #include "PWGCF/DataModel/FemtoDerived.h"
 
 namespace o2::analysis::femtoDream
 {
 
-enum kDetector {
-  kTPC = 0,
-  kTPCTOF = 1,
-  kNdetectors = 2
-};
+enum kDetector { kTPC = 0,
+                 kTPCTOF = 1,
+                 kNdetectors = 2 };
 
-/// internal function that returns the kPIDselection element corresponding to a specifica n-sigma value
-/// \param nSigma number of sigmas for PID
-/// \param vNsigma vector with the number of sigmas of interest
-/// \return kPIDselection corresponding to n-sigma
+/// internal function that returns the kPIDselection element corresponding to a
+/// specifica n-sigma value \param nSigma number of sigmas for PID \param
+/// vNsigma vector with the number of sigmas of interest \return kPIDselection
+/// corresponding to n-sigma
```

Comment:
Keep both `\param` and the `\return` immediately after the `///` for proper doxygen generation

Reply:
Fixed.